llvm-libcxxabi-2017-01-27/test-libfuzzer.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright 2016 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+set -x
+. $(dirname $0)/../common.sh
+
+rm -rf $CORPUS
```

Comment:
We probably also need to remove any fuzzing logs in these scripts.

Reply:
Done!